content/pascal/concepts/functions/functions.md

```diff
@@ -0,0 +1,160 @@
+---
+Title: 'Functions'
+Description: 'Pascal allows for the creation of functions to modularize code and promote code reuse.'
+Subjects:
+ - 'Computer Science'
+ - 'Software Development'
+Tags:
```

Comment:
Tags need to come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If you want to add a new tag, yo need to add it to that file in this PR.